callbacks.go

```diff
@@ -183,3 +185,17 @@ func (exec *sidecarExecutor) Shutdown(driver executor.ExecutorDriver) {
 func (exec *sidecarExecutor) Error(driver executor.ExecutorDriver, err string) {
 	log.Info("Got error message:", err)
 }
+
+// Check if it should check Sidecar status, assuming enabled by default
+func shouldCheckSidecar(containerConfig *docker.CreateContainerOptions) bool {
+	if _, ok:= containerConfig.Config.Labels["SidecarDiscover"]; !ok {
```

Comment:
We should probably check the _value_ of that label as well. Someone might flip it by changing `false` to `true`